news/13795-transaction-hooks

```diff
@@ -0,0 +1,19 @@
+### Enhancements
+
+* Add the ability to define pre- and post-transaction hooks. (#14754)
+
+### Bug fixes
+
+* <news item>
+
+### Deprecations
+
+* <news item>
```

Comment:
Needs entries for the deprecations. See example syntax at https://github.com/conda/conda/blob/1ec58ac92abf05907786c329c65989d79ced6c2e/news/14750-prefixdata#L12-L22